package.json

```diff
@@ -4,17 +4,22 @@
   "description": "SBVR type definitions.",
   "main": "out",
   "scripts": {
-    "lint": "resin-lint src test",
+    "lint": "resin-lint test && resin-lint --typescript src",
     "pretest": "npm run lint && npm run prepare",
     "test": "mocha",
     "prepublish": "require-npm4-to-publish",
-    "prepare": "coffee --output out --compile src"
+    "prepare": "npx tsc",
```

Comment:
Why is npx needed? Isn't `tsc` already installed?

Reply:
It probably just came from the fact I normally run tsc with `npx tsc` so typed it from habit

Reply:
fixed